static/sidebar.js

```diff
@@ -0,0 +1,23 @@
+Polymer({
+  is: "ufo-sidebar",
+  ready: function() {
+    this.pages = [
+    {"href": "/", "text": "Home"},
+    {"href": "/user", "text": "Users"},
```

Comment:
:+1:  Looks good to me as well.  Also (can do this later), should we start to think about having some sort of functional testing for the UI?  In this case, verify that clicking one of the links actually goes to that page?  So, if we change one of the handlers in the backend, the test will remind us to change the links here in the UI as well.